internal/config/colorscheme.go

```diff
@@ -96,7 +97,17 @@ func ParseColorscheme(text string) (map[string]tcell.Style, error) {
 			continue
 		}
 
-		matches := parser.FindSubmatch([]byte(line))
+		matches := includeParser.FindSubmatch([]byte(line))
+		if len(matches) == 2 {
+			include := string(matches[1])
+			err = LoadColorscheme(include)
```

Comment:
What happens if the included colorscheme itself includes the original colorscheme?

(It's a  rhetorical question. I already know what happens.)

Reply:
:see_no_evil: 

Once again, thank you! :+1: